registry/tests/appRoutes.spec.ts

```diff
@@ -240,15 +275,44 @@ describe(`Tests ${example.url}`, () => {
 
                 expect(response.body).deep.equal(expectedRoute);
             } finally {
-                await request.delete(example.url + id);
+                await request.delete(example.url + id).expect(204);
+            }
+        });
+
+        it('should create record with existed domainId', async () => {
+            const domainId = await createRouterDomain(example.routerDomain);
```

Comment:
Same here, I assume we need to cleanup this too

Reply:
done